plugins/net_plugin/net_plugin.cpp

```diff
@@ -3983,6 +4015,29 @@ namespace eosio {
       return fc::json::from_string(s).as<T>();
    }
 
+   size_t net_plugin_impl::parse_connection_limit( const std::string& limit_str) {
+      std::istringstream in(limit_str);
+      fc_dlog( logger, "parsing connection endpoint with locale ${l}", ("l", std::locale("").name()));
```

Comment:
Also include the `limit_str` in the debug log output.

Reply:
Added.